src/tdsops.f90

```diff
@@ -46,7 +46,7 @@ module m_tdsops
                                       der2nd, der2nd_sym, &
                                       stagder_v2p, stagder_p2v, &
                                       interpl_v2p, interpl_p2v
-      integer :: nrank, nproc, pnext, pprev, n, n_blocks
+      integer :: nrank, nproc, pnext, pprev, n, n_blocks, dir
```

Comment:
I think `dir` can be removed again here.

Reply:
Actually we make use of this. When we're in transeq_component, we don't really know what direction we work at, but we need to request a temporary block. `dir` emmber of the `dirps_t` helps us for requesting the right direction.
Also, now that we have a `dir` info in `dirps_t`, we can do a sanity check in `tds_solve` for example. Last commit added a simple sanity check, which actually helped me fixing a minor bug!

Reply:
ah great, I didn't check in details but just assumed it wasn't needed because that was the only change in this file.